io.gpodder.gPodder.json

```diff
@@ -0,0 +1,77 @@
+{
+  "app-id": "io.gpodder.gPodder",
```

Comment:
There doesn't seem to be an actual domain name here. We need the app to comply with the flatpak guidelines: http://docs.flatpak.org/en/latest/introduction.html#identifiers